tests/unit/apps/test_flush_units.py

```diff
@@ -0,0 +1,52 @@
+import unittest
+from parameterized import parameterized
+import lusid
+import pathlib
+import tests.unit.apps.test_data.test_transactions as txn_test_data
+import lusidtools.apps.flush_transactions as flush
+import logging
+
+
+class FlushAndFillUnitTests(unittest.TestCase):
```

Comment:
Can this go in the class for [CLI app unit testing?](https://github.com/finbourne/lusid-python-tools/blob/7da8bd54b93c4b41b8c3fd1caeac17bf036e2538/tests/unit/apps/test_command_line_utilities.py)